src/mq.c

```diff
@@ -54,6 +56,9 @@ struct mq_context {
 };
 
 static struct mq_context mq_ctx;
+static const int8_t num_of_headers = MQ_NUM_OF_HEADERS;
+amqp_table_entry_t headers[MQ_NUM_OF_HEADERS];
+amqp_bytes_t worker_queue;
```

Comment:
Idk if it is right to call this worker_queue. A worker queue is intended to distribute tasks among workers (https://www.rabbitmq.com/tutorials/tutorial-two-python.html) and it is not what happens here. Maybe we can just call as queue or current_queue.